common/types.go

```diff
@@ -145,3 +148,12 @@ func (r HTTPStatusError) Error() string {
 func (r HTTPStatusError) Unwrap() error {
 	return r.err
 }
+
+type ListObjectMetadataResult map[string]ObjectMetadata
+
+type ObjectMetadata struct {
+	DisplayName string
```

Comment:
Can you add a comment for DisplayName? Since the types in this file are publicly exposed, we should generally try to document all of them.